phylogenetic/defaults/config.yaml

```diff
@@ -25,6 +25,7 @@ refine:
 
 ancestral:
   inference: "joint"
+  root_sequence: "reference.gb"
```

Comment:
> This commit only adds the new config option to the CI config. It is not used in the default Nextstrain builds because it's not clear whether it's needed.

Based on the commit message ☝️ , did you mean to make this change in `phylogenetic/build-configs/ci/config.yaml`?

Reply:
If keeping, just update the commit message to match the change 😄

Reply:
😄 Keeping, updated msg and squashed all changes by fixup https://github.com/nextstrain/mumps/pull/46/commits/f066ef118fe7e3797e7fcc27a5dec116c72a327c